uninstall.sh

```diff
@@ -248,6 +253,10 @@ if [ "$doUninstall" == true ]; then
 	# 	rm -rf ~/Applications/Vita3K &>> /dev/null
 	# 	rm -rf ~/.local/share/applications/xenia.desktop &>> /dev/null
 	# fi
+	if [[ "$doUninstallVita3K" == true ]]; then
```

Comment:
This doesn't seem correct

Reply:
ugh i need to be better at checking my work. fixed